samples/signer/setup.sh

```diff
@@ -0,0 +1,167 @@
+#!/bin/bash
+
```

Comment:
Could you add a short description of the purpose of this file?

Reply:
Done.